contrib/caldav/calcurse-caldav.py

```diff
@@ -53,13 +59,20 @@ def calcurse_wipe():
 
 
 def calcurse_import(icaldata):
-    p = subprocess.Popen([calcurse, '-i', '-', '--dump-imported', '-q',
-                          '--format-apt=%(hash)\\n',
-                          '--format-recur-apt=%(hash)\\n',
-                          '--format-event=%(hash)\\n',
-                          '--format-recur-event=%(hash)\\n',
-                          '--format-todo=%(hash)\\n'],
-                         stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+    command = [
+        calcurse,
+        '-i', '-',
+        '--filter-type', sync_filter,
```

Comment:
Minor nit: This should go after, `--dump-imported` and `-q`, just before the formatting options.